src/assets/scss/mts/modules/key-value-list/_key-value-list.scss

```diff
@@ -24,6 +24,8 @@
   &__value {
     @include bold-19;
     padding-left: $gutter-half;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
```

Comment:
What elements use this class out of interest?

Reply:
By a quick glance it looks to be recording events, risk assessment and group tester tables.